### PR TITLE
Changelog for v1.9.3

### PR DESCRIPTION
**Pull request by guillep2k**
As title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add change title notification for issues (#8064)
+  * Run CORS handler first for /api routes (#7967) (#8053)
+  * Evaluate emojis in commit messages in list view (#8044)
+  * Fix failed to synchronize tags to releases for repository (#7990) (#7994)
+  * Fix adding default Telegram webhook (#7972) (#7992)
+  * Abort synchronization from LDAP source if there is some error.  (#7965)
+* ENHANCEMENT
+  * Keep blame view buttons sequence consistent with normal view when viewing a file (#8007) (#8009)
+* MISC
+  * Bugfix/deformed emoji in commit message (#8071)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Can you move this under Bugfixes? And change it to `Fix deformed emoji in commit message (#8071)`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ This changelog goes through all the changes that have been made in each release
...
+* BUGFIXES
+  * Fix go get from a private repository with Go 1.13 (#8100)
+  * Strict name matching for Repository.GetTagID() (#8082)
+  * Avoid ambiguity of branch/directory names for the git-diff-tree command (#8070)
+  * Add change title notification for issues (#8064)
+  * Run CORS handler first for /api routes (#7967) (#8053)
+  * Evaluate emojis in commit messages in list view (#8044)
+  * Fix failed to synchronize tags to releases for repository (#7990) (#7994)
+  * Fix adding default Telegram webhook (#7972) (#7992)
+  * Abort synchronization from LDAP source if there is some error.  (#7965)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Abort synchronization from LDAP source if there is some error (#7965)
```

File: CHANGELOG.md (hunk 1 above)
Comment (guillep2k) [reply CHANGELOG.md]: Done

**Review by techknowlogick — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by guillep2k** [comment]
The Changelog.md was merged into `release/v1.9` ... should I PR for `master` too?

**Pull request merged**